### ISSUE TITLE
Form "required" questions are not accessible

### ISSUE THREAD

**Issue by davidbenque** [xwiki-labs/cryptpad#2194]
Required questions in forms do not appear as such to screen reader users. Two changes would improve this situation:

- Use the "required" HTML attribute on the element. (investigate how to proceed for questions such as polls where the response is not made through `<input>`, `<select>`, or `<textarea>` elements since [the attribute seems limited to those](https://www.w3schools.com/TAGS/att_required.asp)).
- Link the list of missing required questions at the bottom of the form to the questions, so that pressing enter on one moves focus to the question.

**Comment by zuzanna-maria** [comment]
Completed with #2260 as of 2026.5
